From 0f009ea747069de80ef40db664a9f4b62bac186f Mon Sep 17 00:00:00 2001
From: Victor Bo <bvoid@yandex.ru>
Date: Sat, 28 Mar 2020 18:54:56 +0200
Subject: [PATCH 2/4] rework dark themes

	modified:   themes/dark/charcoalblack/res/values-night/colors.xml
	modified:   themes/dark/midnightblue/res/values-night/colors.xml
---
 themes/dark/charcoalblack/res/values-night/colors.xml | 4 ++--
 themes/dark/midnightblue/res/values-night/colors.xml  | 4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/themes/dark/charcoalblack/res/values-night/colors.xml b/themes/dark/charcoalblack/res/values-night/colors.xml
index cfee64c..6a52a6e 100644
--- a/themes/dark/charcoalblack/res/values-night/colors.xml
+++ b/themes/dark/charcoalblack/res/values-night/colors.xml
@@ -30,13 +30,13 @@
      <color name="primary_material_dark">#ff0f1012</color> <!-- material_grey_900 -->
     <color name="primary_material_settings">#ff0f1012</color> <!-- material_grey_900 -->
     <color name="primary_dark_material_dark">@android:color/black</color>
-    <color name="primary_dark_material_settings">#ff151618</color> <!-- material_grey_850 -->
+    <color name="primary_dark_material_settings">#ff0f1012</color> <!-- material_grey_850 -->
     <color name="secondary_material_settings">#ff1a1b1e</color> <!-- material_grey_800 -->
     <color name="tertiary_material_settings">#ff202126</color>
     <color name="quaternary_material_settings">#ffdadce0</color> <!-- material_grey_200 -->
 
     <color name="material_grey_900">#ff0f1012</color>
-    <color name="material_grey_850">#ff151618</color>
+    <color name="material_grey_850">#ff0f1012</color>
     <color name="material_grey_800">#ff1a1b1e</color>
     <color name="material_grey_600">#ff202126</color>
     <color name="material_grey_300">#ff3c4043</color>
diff --git a/themes/dark/midnightblue/res/values-night/colors.xml b/themes/dark/midnightblue/res/values-night/colors.xml
index c4be946..282fe89 100644
--- a/themes/dark/midnightblue/res/values-night/colors.xml
+++ b/themes/dark/midnightblue/res/values-night/colors.xml
@@ -30,13 +30,13 @@
     <color name="primary_material_dark">#ff0d2029</color> <!-- material_grey_900 -->
     <color name="primary_material_settings">#ff0d2029</color> <!-- material_grey_900 -->
     <color name="primary_dark_material_dark">#ff09181e</color>
-    <color name="primary_dark_material_settings">#ff182a33</color> <!-- material_grey_850 -->
+    <color name="primary_dark_material_settings">#ff0d2029</color> <!-- material_grey_850 -->
     <color name="secondary_material_settings">#ff1b2e38</color> <!-- material_grey_800 -->
     <color name="tertiary_material_settings">#ff1e343f</color>
     <color name="quaternary_material_settings">#ff3c4043</color> <!-- material_grey_200 -->
 
     <color name="material_grey_900">#ff0d2029</color>
-    <color name="material_grey_850">#ff182a33</color>
+    <color name="material_grey_850">#ff0d2029</color>
     <color name="material_grey_800">#ff1b2e38</color>
     <color name="material_grey_600">#ff1e343f</color>
     <color name="material_grey_300">#ff9db3be</color>
-- 
2.20.1

